instrument/Facilities.xml

```diff
@@ -50,6 +50,14 @@
       <connection name="event" address="NDXENGINX:10000" listener="ISISLiveEventDataListener" />
     </livedata>
   </instrument>
+  
+  <instrument name="DENEX" shortname="DENEX" >
```

Comment:
Denex is the name of the detector rather than the instrument, and I guess this is here only for testing anyway.
However, maybe we should add HZB as a facility and give it a proper instrument entry for the `V20` instrument?
The broker address in that case should be `192.168.1.80:9092`.

Reply:
Looks like there are already entries for V20, HZB is already present, as are some entries for V20. I would suggest adding a new one: `V20_DENEX`.

Reply:
@LamarMoore If you want to leave an entry with `hinata` as the address for testing then it should probably go in the `TESTLIVE` facility.
`V20` instrument should be moved to `HZB` facility and the broker address changed to `192.168.1.80:9092`.